packages/util-fns/obj_util/index.js

```diff
@@ -0,0 +1,10 @@
+function mapObj(obj, keyTransformFn = x => x, valTransformFn = x => x) {
+  return Reflect.ownKeys(obj).reduce((acc, key) => {
```

Comment:
Since this is a utility function, are you sure that you are only going to get objects?

What about 

```js
mapObj(null) 
mapObj(undefined)
```
because `Reflect.ownKeys` is going to throw TypeError exception.

Reply:
Good point; will write some guards